.changeset/legal-lands-warn.md

```diff
@@ -0,0 +1,43 @@
+---
+"@spectrum-css/contextualhelp": major
+---
+
+#### S2 migration for contextual help
+
+This migrates the `contextual help` component to the latest Spectrum 2 designs. Custom properties have been remapped and added per the design specification.
+
+Typographic and color tokens have been updated per design specifications.
+
+All existing popover positioning varations are supported.
+
+##### New custom properties
+
+`--spectrum-contextual-help-body-color`
+`--spectrum-contextual-help-body-line-height`
+`--spectrum-contextual-help-body-sans-serif-font-family`
+`--spectrum-contextual-help-body-sans-serif-font-style`
+`--spectrum-contextual-help-body-sans-serif-font-weight`
+`--spectrum-contextual-help-title-color`
+`--spectrum-contextual-help-title-font-style`
+`--spectrum-contextual-help-title-font-weight`
+`--spectrum-contextual-help-title-line-height`
+`--spectrum-contextual-help-title-sans-serif-font-family`
+
+##### New mods
+
+`--mod-contextual-help-body-line-height`
+`--mod-contextual-help-body-sans-serif-font-family`
+`--mod-contextual-help-body-sans-serif-font-style`
+`--mod-contextual-help-body-sans-serif-font-weight`
+`--mod-contextual-help-body-size`
+`--mod-contextual-help-title-color`
+`--mod-contextual-help-title-font-style`
+`--mod-contextual-help-title-font-weight`
+`--mod-contextual-help-title-line-height`
+`--mod-contextual-help-title-sans-serif-font-family`
+`--highcontrast-contextual-help-heading-color`
+`--highcontrast-contextual-help-title-color`
+
+##### Removed mods
+
+`--highcontrast-contextual-help-heading-color`
```

Comment:
I'm not sure I'd consider our highcontrast properties as API or modifiers since they're for internal-use only. Maybe we remove this from our changelog?